<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditRecv">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>reading in....</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSend">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInitScanner">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>InitScanner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTriggerPicture">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TriggerPicture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCaptureTarget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CaptureTarget</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCaptureTargetError">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CaptureTarget ParaError</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCaptureTargetResponseSuccess">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CaptureTarget Response Success</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCaptureTargetResponseFail">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CaptureTarget Response Fail</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteTempl">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DeleteTempl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddTempl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AddTempl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearTempl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearTempl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetSelectedTempl">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GetSelectedTempl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetAllTempl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GetAllTemplate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
